__tests__/app.test.js

```diff
@@ -117,4 +117,53 @@ describe("GET: /api/users", () => {
       .expect(404)
     })
 })
-      
\ No newline at end of file
+
+describe('6. PATCH /api/articles/:article_id  ', () => {
+test("200: responds with updated votes", () => {
+  const newVote = {
+    inc_vote: 3
```

Comment:
A small note but the ticket specified the key should be called inc_vote**s** not inc_vote